js/quote_tool.js

```diff
@@ -1 +1,64 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+// Qouting tool logic should be placed here.
+
+// Remember to import consts.js if you will use this file 
+
+/* calculateCost calculates the value of the package that the user prefers 
+ * by giving a value of each one of the options selected by the user
+ * 
+ * The expected objet structure is :
+ * {
+ *  numberOfPeople: integer,
+ *  numberOfNigth: integer,
+ *  trasnport: object, 
+ *  tour: object,
+ *  hike: object,
+ * } 
+ * 
+ * trasnport: {
+ *  medNec: boolean,
+ *  necCap: boolean,
+ *  capNec: boolean,
+ *  necMed: boolean,
+ * }
+ * tour: {
+ *  aguacatePlayasoledad: boolean,
+ *  sapzurroLamiel: boolean,
+ * }
+ * hike: {
+ *  elCieloEltrebol: boolean,
+ *  elParaiso: boolean,
+ *  laCoquerita: object,
+ * }
+ * laCoquerita: {
+ *  hike: boolean,
+ *  guide: boolean,
+ * }
+ */
+function calculateCost(){
+    let total = 0;
+    // PeopleAndNigth It contains the number of people multiplied the number of 
+    //nights and then multiply the result by the heat of each night.
```

Comment:
"and then multiply the result by the heat of each night." is not pertinent to this part of the code